src/security.md

```diff
@@ -9,8 +9,11 @@ the risk of introducing a vulnerability.
 
 ## Reporting vulnerabilities
 
-In the rare event that you find a vulnerability in Dart itself,
-contact us at [https://goo.gl/vulnz](https://goo.gl/vulnz).
+To report a security issue, please use [https://g.co/vulnz](https://g.co/vulnz).
+We use https://g.co/vulnz for our intake, and do coordination and disclosure
+here on GitHub (including using GitHub Security Advisory). The Google Security
```

Comment:
Remove `here` because `here` is dart.dev in this context (we want to publish on GitHub because that enables automated publishing to other DBs later).